doc/nrf/releases_and_maturity/migration/migration_bicr_nRF54H20.rst

```diff
@@ -0,0 +1,478 @@
+The JSON configuration provides explicit control over all parameters.
+
+See the following example:
+
+.. code-block:: json
+
+   {
+     "lfosc": {
+       "source": "LFXO",
+       "lfxo": {
+         "mode": "CRYSTAL",
+         "accuracyPPM": 20,
+         "startupTimeMs": 600,
+         "builtInLoadCapacitancePf": 15,
+         "builtInLoadCapacitors": true
+       }
+     }
+   }
+
+Source
+------
+
+The ``source`` option in JSON can be one of the following:
+
+* ``LFXO``, when an external crystal oscillator is in place.
+* ``LFRC``, when an external Crystal Oscillator is not in place.
+
+This means that the device can use either ``LFRC`` or ``SYNTH`` as clock sources.
+
+LFRC autocalibration configuration
+==================================
+
+This DTS option configures the ``BICR`` → ``LFOSC.LFRCAUTOCALCONFIG`` register.
+
+.. code-block:: dts
+
+   lfrc-autocalibration = <20 40 3>;
+
+The three values provided in the BICR DTS format correspond to the LFRC autocalibration configuration:
+
+* ``temp-interval``: Specifies the interval between temperature measurements, expressed in 0.25-second increments.
+* ``temp-delta``: Defines the temperature change, in 0.25-degree Celsius steps, that triggers a calibration event.
+* ``interval-max-count``: Indicates the maximum number of measurement intervals allowed between calibrations, regardless of temperature variations.
+
+In the DTS property, each variable is mapped as follows:
+
+* A: Temperature measurement interval (``temp-interval``)
+* B: Temperature delta for calibration (``temp-delta``)
+* C: Maximum intervals between calibrations (``interval-max-count``)
+
+Use these parameters to precisely control LFRC autocalibration behavior.
+
+Datasheet register field
+========================
+
++---------------------+------------------------------------+-------------------------------+
+| DTS value           | JSON variable                      | JSON value                    |
++=====================+====================================+===============================+
+| TEMPINTERVAL (A)    | tempMeasIntervalSeconds            | 5                             |
+| TEMPDELTA (B)       | tempDeltaCalibrationTriggerCelsius | 10                            |
+| INTERVALMAXNO (C)   | maxMeasIntervalBetweenCalibrations | 3                             |
+| ENABLE              | calibrationEnabled                 | -                             |
++---------------------+------------------------------------+-------------------------------+
+
+If ``lfrc-autocalibration`` is set in DTS, the ``ENABLE`` field in ``BICR`` → ``LFOSC.LFRCAUTOCALCONFIG`` will be set automatically.
+
+LFRC Autocalibration is not configured in the JSON configuration files for the DK, as the default values (``4``, ``0.5`` and ``2``) will be good enough for most use-cases.
+However, the DTS example above would translate to JSON as such:
```

Comment:
```suggestion
LFRC autocalibration is not configured in the JSON configuration files for the DK, as the default values (``4``, ``0.5``, and ``2``) are good enough for most use-cases.
However, the previous DTS example would translate to JSON as such:
```
SoC in some places and DK in others. Might be good to use just one of them throughout the doc.

Reply:
The thing is that the guide is for the SoC, but it takes the DK files as an example. There will be mentions of both. I'm going through both to see if they are mentioned where originally intended.